pom.xml

```diff
@@ -934,7 +934,7 @@
         <msf4j.version>2.1.0</msf4j.version>
         <org.wso2.carbon.caching.version>1.0.0</org.wso2.carbon.caching.version>
         <org.wso2.carbon.deployment.version>5.0.0</org.wso2.carbon.deployment.version>
-        <org.wso2.carbon.identity.event.handler.version>1.0.2</org.wso2.carbon.identity.event.handler.version>
+        <org.wso2.carbon.identity.event.handler.version>1.0.7-SNAPSHOT</org.wso2.carbon.identity.event.handler.version>
```

Comment:
This needs to be merged with the released version of notification-event-handler, after its PR merged.